sass/_product_display.scss

```diff
@@ -193,28 +192,38 @@
 				}
 			}
 
-			.download-link {
+			.external-link {
 				display: inline-flex;
 				align-items: center;
-				color: #f77d06;
 				text-decoration: none;
 				cursor: pointer;
 				letter-spacing: 0.08em;
 
 				&:hover {
-					color: darkorange;
+					color: #ffa348;
+					text-decoration: underline;
 
-					svg {
-						fill: darkorange;
+					.external-icon {
+						filter: invert(77%) sepia(31%) saturate(1690%) hue-rotate(324deg) brightness(102%) contrast(102%);
```

Comment:
We won't use back & white icon ever. Convert the image so that we don't need to apply the filter every time.

Reply:
This looks nice.
https://stackoverflow.com/a/65147574/640650

Reply:
it also says:

> It doesn't work if SVG is saved as a file and displayed with <img> tag.

Reply:
Ok. If you've alrady done. That's fine. I thought you want to do it in runtime.